packages/ui/src/components/ChatList.tsx

```diff
@@ -242,164 +150,273 @@ export function ChatList({
     }
   ).current;
 
-  const FILTER_HEIGHT =
-    getTokenValue('$6xl', 'size') + getTokenValue('$4xl', 'size');
-  const HEADER_HEIGHT = getTokenValue('$4xl', 'size');
-  const SNAP_THRESHOLD = FILTER_HEIGHT / 2;
+  const handlePressTryAll = useCallback(() => {
+    setActiveTab('all');
+  }, [setActiveTab]);
 
-  const { top } = useSafeAreaInsets();
-  const filterStyle = useAnimatedStyle(() => {
-    const translateY = interpolate(
-      scrollY.value,
-      [-FILTER_HEIGHT, 0],
-      [0, -FILTER_HEIGHT],
-      Extrapolation.CLAMP
-    );
+  const handlePressClear = useCallback(() => {
+    setSearchQuery('');
+  }, []);
 
-    return {
-      height: FILTER_HEIGHT,
-      transform: [{ translateY: filterVisible.value ? 0 : translateY }],
-      position: 'absolute',
-      top: top + HEADER_HEIGHT,
-      left: 0,
-      right: 0,
-      zIndex: 40,
-    };
-  });
+  return (
+    <>
+      <ChatListFilters
+        query={searchQuery}
+        onQueryChange={setSearchQuery}
+        activeTab={activeTab}
+        onPressTab={setActiveTab}
+        isOpen={showFilters}
+      />
+      {searchQuery !== '' && !displayData[0]?.data.length ? (
```

Comment:
Should this be `searchQuery.trim() !== ''`?

If that's a bug, I think `useFilteredChats` could give more info here to avoid similar bugs – e.g. return `{isSearching: boolean, hasResults: boolean, displayData: ReturnValueOfCurrentImpl}` then just check `isSearching` here